resources/ts/Components/Integrations/Detail/CredentialsAuthClients.tsx

```diff
@@ -52,7 +52,9 @@ export const CredentialsAuthClients = ({
   ];
 
   const handleKeyVisibilityUpgrade = () =>
-    router.post(`/integrations/${id}/upgrade`, { keyVisibility });
+    router.post(`/integrations/${id}/upgrade`, {
+      keyVisibility: KeyVisibility.all,
```

Comment:
This needs to be `v2`  because you request an upgrade to `v2`. It is true this will result in a key visibility of `all` on the integration, but the upgrade request is from `v1` to `v2` so in this context it needs to be `v2`